bridges/opentracing/lib/opentelemetry/bridge/opentracing/tracer.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # Tracer provides a means of referencing
+      # an OpenTelemetry::Tracer as a OpenTracing::Tracer
+      class Tracer
+        HTTP_TEXT_FORMAT = OpenTelemetry::DistributedContext::Propagation::HTTPTextFormat.new
+        BINARY_FORMAT = OpenTelemetry::DistributedContext::Propagation::BinaryFormat.new
+
+        def active_span
+          OpenTelemetry.tracer.current_span
+        end
+
+        def start_active_span(operation_name,
```

Comment:
This should optionally take a block and evaluate the block within the context of the newly created span. Also, this method should return a scope.